lizrd/support/logging.py

```diff
@@ -329,49 +431,46 @@ def log_plot(figure: plotly.graph_objs.Figure, title: str, series: str, iteratio
 def get_logger(args, model, VOCAB_SIZE):
     timestamp = make_concise_datetime()
     unique_timestamp = f"{timestamp}{secrets.token_urlsafe(1)}"
-    if args.use_neptune:
-        run = neptune.init_run(
-            project=args.project_name,
-            tags=args.tags,
-            name=f"{args.name} {tags_to_name(args.tags)} {unique_timestamp}",
-        )
-        run["args"] = vars(args)
-        run["working_directory"] = os.getcwd()
-        run["config"].upload(args.path_to_entry_config)
-        all_config_paths = args.all_config_paths.split(",")
-        run["all_configs"].upload_files(all_config_paths)
-
-        args.model_n_params = count_parameters(model, args, VOCAB_SIZE)
-        return NeptuneLogger(run, args)
-
-    elif args.use_clearml:
-        task = Task.init(
-            project_name=args.project_name,
-            task_name=f"{args.name} {tags_to_name(args.tags)} {unique_timestamp}",
-        )
-        task.connect(vars(args))
-        if args.tags:
-            task.add_tags(args.tags)
-        logger = ClearMLLogger(task, args, model, VOCAB_SIZE)
-        return logger
-    elif args.use_wandb:
-        wandb.init(
-            entity=args.wandb_entity,
-            project=args.project_name,
-            name=f"{args.name} {tags_to_name(args.tags)} {unique_timestamp}",
-            tags=args.tags,
-            config=vars(args),
-        )
-        # define our custom x axis metric
-        wandb.define_metric("train/step")
-        # set all other train/ metrics to use this step
-        wandb.define_metric("*", step_metric="train/step")
-        return WandbLogger(wandb, args)
+    if args.logger_types == "":
+        logger_types = []
     else:
-        print(
-            "No logger specified! either args.use_neptune / args.use_clearml / args.use_wandb must be True"
-        )
-        raise NotImplementedError
+        logger_types = args.logger_types.split(",")
+    initialized_loggers = []
+    for logger_type in logger_types:
```

Comment:
Maybe let's add validation that doesn't allow specifying the same type twice? e.g. "neptune,stdout,neptune"

Reply:
Good catch!